### PR TITLE
Fixup benches after dir re-org

### PR DESCRIPTION
**Pull request by gilescope**
Close but no cigar. Missed out the category.

### CODE REVIEW

File: scripts/benchmarks-ci.sh [hunk 1]
```diff
@@ -2,10 +2,11 @@
 
 steps=50
 repeat=20
-chainName=$1
+category=$1
+runtimeName=$2
 
-benhcmarkOutput=./parachains/runtimes/$chainName/src/weights
-benhcmarkChainName="$chainName-dev"
+benhcmarkOutput=./parachains/runtimes/$category/$runtimeName/src/weights
```
[the remark this answers is not in the document]
Comment (chevdor) [reply scripts/benchmarks-ci.sh:8]: And I think this fix need to be propagated wherever we use `benhcmarkOutput` atm.

**Review by chevdor — CHANGES_REQUESTED** [review]
typos

File: scripts/benchmarks-ci.sh [hunk 2]
```diff
@@ -2,10 +2,11 @@
...
+benhcmarkRuntimeName="$runtimeName-dev"
```
[the remark this answers is not in the document]
Comment (gilescope) [reply scripts/benchmarks-ci.sh:9]: good spot

File: scripts/benchmarks-ci.sh [hunk 3]
```diff
@@ -2,10 +2,11 @@
 
 steps=50
 repeat=20
-chainName=$1
+category=$1
+runtimeName=$2
 
-benhcmarkOutput=./parachains/runtimes/$chainName/src/weights
-benhcmarkChainName="$chainName-dev"
+benchmarkOutput=./parachains/runtimes/$category/$runtimeName/src/weights
```
Comment (chevdor) [inline scripts/benchmarks-ci.sh:8]: This variable is either unused and not required or we miss a rename where it is used.

**Review by chevdor — CHANGES_REQUESTED** [review]

File: scripts/benchmarks-ci.sh (hunk 3 above)
Comment (chevdor) [reply scripts/benchmarks-ci.sh:8]: I take it back, just saw it.

**Review by chevdor — APPROVED** [review]

**Review by NachoPal — APPROVED** [review]

**Review by alvicsam — APPROVED** [review]

**Review by sergejparity — CHANGES_REQUESTED** [review]

File: .gitlab-ci.yml [hunk 4]
```diff
@@ -211,9 +211,9 @@ benchmarks:
   <<:                              *collect-artifacts
   <<:                              *benchmarks-refs
   script:
-    - ./scripts/benchmarks-ci.sh statemine > ./artifacts/bench-statemine.log
-    - ./scripts/benchmarks-ci.sh statemint > ./artifacts/bench-statemint.log
-    - ./scripts/benchmarks-ci.sh westmint > ./artifacts/bench-westmint.log
+    - ./scripts/benchmarks-ci.sh assets statemine > ./artifacts/bench-statemine.log
+    - ./scripts/benchmarks-ci.sh assets statemint > ./artifacts/bench-statemint.log
+    - ./scripts/benchmarks-ci.sh assets westmint > ./artifacts/bench-westmint.log
```
[the remark this answers is not in the document]
Comment (chevdor) [reply .gitlab-ci.yml:216]: If that's ok with you @sergejparity I would prefer to merge this PR as it is and asap so the benchmarks work again.
We can then line up another PR to move the scripts.
I have opened [an issue](https://github.com/paritytech/cumulus/issues/1320) to track the move.

[the remark this answers is not in the document]
Comment (sergejparity) [reply .gitlab-ci.yml:216]: Ok. No problem.

File: scripts/benchmarks-ci.sh [hunk 5]
```diff
@@ -24,7 +25,7 @@ pallets=(
 for p in ${pallets[@]}
```
[the remark this answers is not in the document]
Comment (sergejparity) [reply scripts/benchmarks-ci.sh:25]: inside `./parachains/runtimes/assets/` those folders exists. So it's not needed.

**Review by sergejparity — APPROVED** [review]

**Review by hbulgarini — APPROVED** [review]
lgtm.

**Review by joepetrowski — APPROVED** [review]

**Pull request merged**